tests/unit/amo/test_installAddon.js

```diff
@@ -298,6 +309,7 @@ describe(__filename, () => {
         });
       });
 
+      // SKIP (is that even used ?)
```

Comment:
Not only is it not used, and I don't even think it works. I don't think this test is valid.